lib/core/src/lib/viewer/components/viewer.component.ts

```diff
@@ -474,7 +478,10 @@ export class ViewerComponent<T> implements OnDestroy, OnInit, OnChanges {
     }
 
     getDisplayFileName(): string {
-        const fullName = (this.fileNameWithoutExtension || '') + (this.fileExtension || '');
+        const title = this.title;
+        const fileName = (this.fileNameWithoutExtension || '') + (this.fileExtension || '');
```

Comment:
Can we get some UX input on that please? Right now if we will have both title and file name which may be long - they will be just cut and left with first part of title + .... + file extension which technically may look fine but it would be good to get the UX point on that, also maybe it would be good idea to make the title display optional/configurable? I'm not quite sure we want to have it in ACA/ADW

Reply:
@MichalKinas  I am closing this PR, final result is visible here:
https://github.com/Alfresco/alfresco-ng2-components/pull/11026

I attached screens